test/functional/mempool_persist.py

```diff
@@ -69,13 +69,19 @@ def run_test(self):
         assert_equal(len(self.nodes[0].getrawmempool()), 5)
         assert_equal(len(self.nodes[1].getrawmempool()), 5)
 
+        total_fee_old = self.nodes[0].getmempoolinfo()['total_fee']
```

Comment:
Perhaps could do a check here that `total_fee_old` is a positive integer?

Reply:
Could also be checked via `CHECK_NONFATAL` on the server side.